src/watts/results.py

```diff
@@ -50,9 +50,9 @@ def move_files(self, dst: PathLike):
         """
 
         dst_path = Path(dst)
-        # Move input/output files and change base -- note that trying to use the
-        # Path.replace method doesn't work across filesystems, so instead we use
-        # shutil.move
+#Move input / output files and change base-- note that trying to use the
+#Path.replace method doesn't work across filesystems, so instead we use
+#shutil.move
```

Comment:
Not sure what happened here -- can you move this back to where it was before?

Reply:
ok